repo_id,repo_full_name,file,file_sha1,match
242373353,mhaskar/CVE-2020-8813,Cacti-preauth-rce.py,3a429de135f155ff0d8b4cc528c8ef877d46c77c,CVE-2020-8813
242373353,mhaskar/CVE-2020-8813,README.md,b85910e25dd28a130e3d5d56b5dbcc0bcaee6486,CVE-2020-8813
242373353,mhaskar/CVE-2020-8813,Cacti-postauth-rce.py,ecdf5b093a56992aeea2a0690277dd58f92f9961,CVE-2020-8813
242373353,mhaskar/CVE-2020-8813,_GITHUB_REPO_METADATA_,,CVE-2020-8813
